doc/configoptions.md

```diff
@@ -5,161 +5,102 @@ in `data/*/*.file_list` (see [file list docu](files.md)).
 
 The options here are environment variables used by `mk_image`. They are typically set in `Makefile`.
 
+Notation: in the reference below, the uppercase FOO is the value of the option
+(environment variable) lowercase foo.
 
 ## General options
 
-- cache=1|2|4
+<!--
+use this to find out actual usage:
+  grep bin/mk_image Makefile | tr ' ' '\n' | sort -u
+-->
 
-  cache the rpms we use, it is actually a bitmask; the most useful value is 4, which
-  will cache the unpacked rpm in a separate directory
+- alternatives=1
 
-- debug=ignore
+  perform a fix-up of update-alternatives links (because...? so that...?)
 
-  ignore noncritical errors
+- disjunct=WORD (eg. `initrd`, `root`)
 
-- debug=account
+  separate WORD from the rest (in what way? so that...?)
```

Comment:
The image will include only files that are not in the image specified with disjunct. It's basically a deduplication
used for inst-sys 'extensions' (like 'bind', 'gdb', ...).

The same thing (but done explicitly by calling the common_tree script) is done in the 'root+rescue' make target to
separate initrd, root, and rescue image into 4 disjuct parts.

Reply:
incorporated